python/ray/serve/tests/test_api.py

```diff
@@ -48,6 +48,42 @@ def function(flask_request):
     assert resp == "POST"
 
 
+def test_backend_user_config(serve_instance):
+    client = serve_instance
+
```

Comment:
let's use SignalActor to synchronize this instead of sending 100 queries

Reply:
@simon-mo Actually I'm not sure about this--I don't think we need any synchronization here, we just want a way of asking each replica if it got updated.  Since replicas are called in round-robin, it's enough to call `handle.remote()` three (or 100) times, and these calls don't have to happen simultaneously.  Does that make sense? 

If we did want to use SignalActor, I'm not sure how that would work, since we don't have a "handle" for each replica, we just have one for the endpoint.

Reply:
Good point.